docs/introduction.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Introduction
+id: introduction
+---
+
+# What is `granary`?
+
+Granary is a job runner for cloud-based geospatial machine learning.
+Its goal is to simplify running models and to track and serve
+the results of predictions. It puts a REST API between you and AWS Batch
+to simplify interactions that otherwise involve repeatedly checking AWS SDK
+documentation.
+
+## What's a `Model`?
+
+A model is a bundle of a human-readable name, some, AWS Batch configuration,
+and an argument validator.
+
+```scala mdoc
+import com.rasterfoundry.granary.datamodel._
+
+import io.circe.syntax._
+
+import java.util.UUID
+
+val model = Model.Create(
+  "A descriptive model name",
+  Validator(().asJson), // a Validator -- more on this below
+  "perfectAccuracyModel:1", // an AWS Batch job definition
+  "veryExpensiveOnDemandQueue" // an AWS Batch job queue
+)
+// What you'll POST to the API to create a model
+model.asJson.spaces2
+```
+
+`model`s in Granary correspond to containers that have been configured to
+run via AWS Batch job definitions. The major difference between using Granary
+and hand rolling `SubmitJob` requests is the `Validator`. To create a model,
+post JSON like what's shown above to `/api/models`.
+
+## What's a `Validator`?
+
+A `Validator` uses JSON Schema to ensure that when you try to run your job,
+you have the correct arguments. For example, in the example above the
+`Validator` is the empty json object `{}`. This means that providing any arguments
+will fail. That model is not a very useful model. We could instead require a green
+band, a red band, and a GeoTIFF location with the following JSON schema:
+
+```json
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://example.com/root.json",
+  "type": "object",
+  "title": "The Root Schema",
+  "required": [
+    "greenBand",
+    "redBand",
+    "tiffLocation"
+  ],
+  "properties": {
+    "greenBand": {
+      "$id": "#/properties/greenBand",
```

Comment:
Is this `#/...` naming pattern for the IDs significant? A JSON Schema thing?

Reply:
I think it's a JSON Schema thing -- it's from the generated schema when I used the editor